libnmstate/schema.py

```diff
@@ -24,50 +24,50 @@
 
 def load(schema_name):
     return yaml.load(
-        pkgutil.get_data('libnmstate', 'schemas/' + schema_name + '.yaml'),
+        pkgutil.get_data("libnmstate", "schemas/" + schema_name + ".yaml"),
```

Comment:
This is against nmstate's [coding guidelines](https://github.com/nmstate/nmstate/blob/master/CONTRIBUTING.md#coding-and-style-guidelines).